src/tb_client_exports.zig

```diff
@@ -11,6 +11,12 @@ pub const std_options = .{
     .logFn = tb.Logging.application_logger,
 };
 
+/// Static backing allocator for clients. Wraps heap.c_allocator to keep maximum compatibility since
+/// as a library, libtbclient is running inside another process's address space.
+var gpa = std.heap.GeneralPurposeAllocator(.{}){
+    .backing_allocator = std.heap.c_allocator,
+};
+
```

Comment:
Maybe also `comptime assert(@sizeOf(gpa.mutex) > 0)` to double-check that it is thread-safe?

Reply:
I've gone with creating the GPA type with `.thread_safe = true`.